From ec5b0ac7093e180ece52c8c116943a6b36b30a6a Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:19 -0400
Subject: [tools] perf: Move perf_call_graph_mode enum from perf.h

Message-id: <1410181752-25631-195-git-send-email-jolsa@redhat.com>
Patchwork-id: 91905
O-Subject: [PATCH RHEL7.1 BZ1134356 194/307] perf tools: Move perf_call_graph_mode enum from perf.h
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 2c83bc08e305e135b954d82596864b7d024fc7fd
Author: Jiri Olsa <jolsa@kernel.org>
Date: Mon May 5 12:46:17 2014 +0200

description
===========
Into util/callchain.h header where all callchain related
structures should be.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index cf8f228..d51a994 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -222,13 +222,6 @@ void pthread__unblock_sigwinch(void);
 
 #include "util/target.h"
 
-enum perf_call_graph_mode {
- CALLCHAIN_NONE,
- CALLCHAIN_FP,
- CALLCHAIN_DWARF,
- CALLCHAIN_MAX
-};
-
 struct record_opts {
  struct target target;
  int      call_graph;
diff --git a/tools/perf/util/callchain.h b/tools/perf/util/callchain.h
index dda4cf8..bde2b0c 100644
--- a/tools/perf/util/callchain.h
+++ b/tools/perf/util/callchain.h
@@ -7,6 +7,13 @@
 #include "event.h"
 #include "symbol.h"
 
+enum perf_call_graph_mode {
+ CALLCHAIN_NONE,
+ CALLCHAIN_FP,
+ CALLCHAIN_DWARF,
+ CALLCHAIN_MAX
+};
+
 enum chain_mode {
  CHAIN_NONE,
  CHAIN_FLAT,
-- 
1.7.1